### DISCUSSION TITLE
Welcome to aws-serverless-data-lake-framework Discussions!

### DISCUSSION THREAD

**Discussion by jaidisido** [aws-solutions-library-samples/data-lakes-on-aws#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mariandumitrascu** [comment]
Hello SDLF,

I think SDLF deserve more attention from the community. It certainty has mine. It is a solid CICD architecture with endless flexibility. Once you get the gist of how it works, it lets you build a data-lake that is from the beginning well architected. 
I would love to contribute.

-Marian

**Reply by kukushking** [reply]
Thanks @mariandumitrascu for feedback! Contributions are very welcome! 

Also you might be interested to have a look at [AWS DataOps Development Kit (DDK)](https://github.com/awslabs/aws-ddk) which is based on AWS CDK IaC.
